Support for tree shaking

Dreaming-Codes opened the pull request:
Now only the parts of the library used are imported into the code

Dreaming-Codes committed:
Support for tree shaking

d0p3t reviewed (changes requested):
What happened to package.json? Did you change the line endings of the file? Please keep line endings the same so the diff only shows actual changes. 

But thank you for this.

Dreaming-Codes committed:
I remade the package.json with the github editor

I remade the package.json with the github editor so maybe the diff doesn't give problems

Dreaming-Codes committed:
In the previous commit I had copied the package.json from the master and not from the dev

Dreaming-Codes commented:
@d0p3t  I really don't know what the problem is even editing with the github editor the problem persists ...

Dreaming-Codes committed:
Maybe now

Dreaming-Codes commented:
#58
